packages/web/stores/user-settings/language.tsx

```diff
@@ -20,6 +20,11 @@ const SUPPORTED_LANGUAGES: MenuDropdownIconItemProps[] = [
     display: "settings.languages.fr",
     iconUrl: "/images/flag-french.png",
   },
+  {
+    value: "pl",
+    display: "settings.languages.pl",
+    iconUrl: "/images/flag-polish.png",
```

Comment:
As mentioned in other comment, the display key here should be hardcoded to "Polish" in polish. Also, we decided to not identify languages with flags of countries. Please update your branch and TypeScript should let you remove `iconUrl` key.